config/services/open-balena-vpn.service

```diff
@@ -26,17 +27,9 @@ WorkingDirectory=/usr/src/app
 EnvironmentFile=/usr/src/app/config/env
 ExecStart=/usr/src/app/bin/start.sh
 ExecStop=-/usr/src/app/bin/stop.sh
-# https://www.freedesktop.org/software/systemd/man/systemd.kill.html#KillMode=
-# only send SIGTERM to master process, send SIGKILL to all remaining processes in the
-# group after TimeoutStopSec has elapsed; this setting prevents VPN worker(s) from
-# restarting after SIGTERM is received and before SIGKILL
-KillMode=mixed
-# https://www.freedesktop.org/software/systemd/man/systemd.service.html#TimeoutStopSec=
-# ensure k8s/terminationGracePeriodSeconds matches
-TimeoutStopSec=${DEFAULT_SIGTERM_TIMEOUT}
 
 Restart=always
 SyslogIdentifier=vpn
 
 [Install]
-WantedBy=basic.target
+WantedBy=multi-user.target
```

Comment:
Does this work with `multi-user.target` again? I know I had to change it to fix things for debian bullseye